src/firefly/js/visualize/ImagePlotCntlr.js

```diff
@@ -339,15 +339,17 @@ export function dispatchUpdateViewSize(plotId,width,height,updateScroll=true,cen
 }
 
 /**
+ * change group lock for zoom and scolling
  *
- * @param plotId
- * @param groupLock
+ * @param {string} plotId is required
+ * @param {boolean} groupLocked,  true to set group lockRelated on
+ * @param {plotGroupAry}
  */
-export function dispatchGroupLocking(plotId,plotGroupId,plotGroupAry,plotGroup,groupLocked) {
+export function dispatchGroupLocking(plotId,plotGroupAry,groupLocked) {
```

Comment:
You still have plotGroupAry.  I don't think you need it, you are getting it from the state in the reducer.  It is not used.